CRM/Queue/Queue.php

```diff
@@ -60,7 +62,17 @@ public function __construct($queueSpec) {
    */
   public function isActive(): bool {
     $status = CRM_Core_DAO::getFieldValue('CRM_Queue_DAO_Queue', $this->_name, 'status', 'name', TRUE);
-    // Note: In the future, we may want to incorporate other data (like maintenance-mode or upgrade-status) in deciding active queues.
+    if ($status === 'active' && \Civi::settings()->get('civi.queue.isActive')) {
```

Comment:
Oh weird @totten I see 

![image](https://github.com/civicrm/civicrm-core/assets/336308/799475f2-cac1-4451-900e-99e35b1b8089)

Reply:
Yep, it looked better for me now too.